docs/manual/getting-started.qmd

```diff
@@ -0,0 +1,77 @@
+```{python}
+#| echo: false
+from pathlib import Path
+
+import IPython.display as display
+
+from odm_sharing.tools.share import share
+
+from common import DATA, SCHEMA, load_csv_md, print_file
+```
+
+# Getting started {#sec-getting-started}
+
+So you want to share your data, but not all of it. This library provides the
+tools for filtering your data before sharing it.
+
+## Data
+
+The input data can either be in CSV or Excel/XLSX format. Here's a CSV example:
+
+`measures.csv`
+```{python}
+#| echo: false
+print_file(DATA)
+```
+
+## Schema
+
+A sharing schema is needed to define the sharing rules. The following example
+schema defines one rule for which columns to select, one for which rows to
+include, and one that binds them together with a recipient organization:
+
+`schema.csv`
+```{python}
+#| echo: false
+print_file(SCHEMA)
+```
+
+See @sec-schemas for more information on schemas.
+
+## CLI
+
+Run `odm-share` from the command line to filter your data using a schema:
+
+```bash
+odm-share schema.csv measures.csv
+```
+
+This will produce one output file per organization (as specified in the schema)
+with the filename pattern `<input_name>-<org>.csv`.
+
+Using the above schema and data examples would result in the following:
+
+`measures-OHRI.csv`
+```{python}
+#| echo: false
+outpath = f'{Path(DATA).stem}-OHRI.csv'
+share(SCHEMA, DATA)
+display.Markdown(load_csv_md(outpath))
+```
+<br/> <!-- needed because `display` doesn't add bottom margin -->
+See @sec-cli for the CLI reference and more examples.
+
+## API
+
+To use the Python API:
+
+```python
+import odm_sharing.sharing as sh
+
+results = sh.extract('schema.csv', 'data.xlsx')
```

Comment:
done

Reply:
I think we should use actual file paths here (./schema.csv for example) in here rather than SCHEMA and DATA since I assume users will be copying this code and running it. Having it as variables makes it a little bit harder for them.

Reply:
I agree, but I wouldn't want to make the entire (internal) path visible to the user. I have solved this issue in the doc-api PR, where I copy the asset files to the local directory before running the code. I can do the same here. Is it ok if I make an issue for it and do it after the PRs? Then I can reuse the code I already made for this in the other PR, and we'll finish these PRs sooner.

Reply:
#22